Microsoft.Recognizers.Text.Number/Extractors/BaseNumberExtractor.cs

```diff
@@ -74,6 +74,47 @@ public virtual List<ExtractResult> Extract(string source)
 
             return result;
         }
+
+        public Regex AddArabicNumberRegex(ArabicType type, string placeholder = @"\D|\b")
```

Comment:
Renaming this method to GenerateArabicNumberRegex is better. And this method should be protected